quantum/nodes/shors/shors.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const util = require('util');
+const snippets = require('../../snippets');
+const errors = require('../../errors');
+const shell = require('../../python').PythonShell;
```

Comment:
```suggestion
const {PythonShellClass} = require('../../python');
const shell = new PythonShellClass();
```
If this node runs separately from other quantum nodes, we'll want to use a fresh `PythonShell` instance here.